doc/src/manual/documentation.md

```diff
@@ -326,7 +327,7 @@ y = MyType("y")
 
 ## Syntax Guide
 
-A comprehensive overview of all documentable Julia syntax.
+A comprehensive overview of Julia's documentation syntax.
```

Comment:
This doesn't have the quite the correct meaning. The original here meant an overview of which Julia syntax is documentable. That is related to the syntax of documentation, but not quite the same thing.

Reply:
```suggestion
This guide provides a comprehensive overview of how to attach documentation to all Julia syntax
constructs for which providing documentation is possible.
```